app/src/main/res/layout-sw600dp/administrator_controls_download_permissions_view.xml

```diff
@@ -71,6 +71,9 @@
         android:layout_marginStart="40dp"
         android:layout_marginEnd="4dp"
         android:checked="@{viewModel.isTopicWifiUpdatePermission}"
+        android:minWidth="@dimen/touch_target_size"
+        android:minHeight="@dimen/touch_target_size"
+        android:contentDescription="@string/administrator_controls_update_on_wifi_title"
```

Comment:
How does this content description read out in context? I suspect we will actually want to create new content descriptions here & elsewhere that are more specific rather than trying to reuse existing strings (that also helps with translating the strings since the contexts differ in these cases).

Reply:
using `Talkback` the description reads `on download and update only on Wi-Fi switch` or `off download and update only on Wi-Fi switch` depending on the state of the switch and similarly for the other switch `on/off automatically update topics switch`

Yes it would be better to create new string resources for content descriptions.
The title strings seemed fine so I didn't create new strings. What do you suggest can be the new strings for `contentDescription`?